From fade93d33ca3ca9890d6d7454e3c9cc61ab82095 Mon Sep 17 00:00:00 2001
From: Myron Stowe <myron.stowe@redhat.com>
Date: Wed, 6 Aug 2014 03:47:11 -0400
Subject: [x86] pci: Fix function definition whitespace

Message-id: <20140806034711.1438.8556.stgit@gir.stowe>
Patchwork-id: 87438
O-Subject: [RHEL7.1 PATCH 29/69] x86/PCI: Fix function definition whitespace
Bugzilla: 1110896
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>
RH-Acked-by: Dean Nelson <dnelson@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1110896
Upstream Status: da5d727c9744e3986c54192a7fbcf10720b6534c

commit da5d727c9744e3986c54192a7fbcf10720b6534c
Author: Bjorn Helgaas <bhelgaas@google.com>
Date:   Fri Jan 24 10:44:42 2014 -0700

    x86/PCI: Fix function definition whitespace

    Consistently put the function type, name, and parameters on one line,
    wrapping only as necessary.

    Signed-off-by: Bjorn Helgaas <bhelgaas@google.com>

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/arch/x86/pci/acpi.c b/arch/x86/pci/acpi.c
index 9eeea64..03bbb77 100644
--- a/arch/x86/pci/acpi.c
+++ b/arch/x86/pci/acpi.c
@@ -218,9 +218,8 @@ static void teardown_mcfg_map(struct pci_root_info *info)
 }
 #endif
 
-static acpi_status
-resource_to_addr(struct acpi_resource *resource,
-   struct acpi_resource_address64 *addr)
+static acpi_status resource_to_addr(struct acpi_resource *resource,
+        struct acpi_resource_address64 *addr)
 {
  acpi_status status;
  struct acpi_resource_memory24 *memory24;
@@ -265,8 +264,7 @@ resource_to_addr(struct acpi_resource *resource,
  return AE_ERROR;
 }
 
-static acpi_status
-count_resource(struct acpi_resource *acpi_res, void *data)
+static acpi_status count_resource(struct acpi_resource *acpi_res, void *data)
 {
  struct pci_root_info *info = data;
  struct acpi_resource_address64 addr;
@@ -278,8 +276,7 @@ count_resource(struct acpi_resource *acpi_res, void *data)
  return AE_OK;
 }
 
-static acpi_status
-setup_resource(struct acpi_resource *acpi_res, void *data)
+static acpi_status setup_resource(struct acpi_resource *acpi_res, void *data)
 {
  struct pci_root_info *info = data;
  struct resource *res;
@@ -435,9 +432,9 @@ static void release_pci_root_info(struct pci_host_bridge *bridge)
  __release_pci_root_info(info);
 }
 
-static void
-probe_pci_root_info(struct pci_root_info *info, struct acpi_device *device,
-      int busnum, int domain)
+static void probe_pci_root_info(struct pci_root_info *info,
+    struct acpi_device *device,
+    int busnum, int domain)
 {
  size_t size;
 
-- 
1.7.1